fgd/brush/trigger/trigger_stick.fgd

```diff
@@ -0,0 +1,11 @@
+@SolidClass base(StaticTargetName, EnableDisable, Toggle) 
+	appliesto(MOMENTUM)
+= trigger_stick : "Conc grenades, stickybombs, and other explosives will stick to or inside this area."
```

Comment:
Adding here (you may have already started this): Rename to `trigger_stick_explosive`